src/interactive.rs

```diff
@@ -456,7 +456,9 @@ async fn execute_query(options: &Options, mut state: &mut repl::State,
                             source: ref error,
                             ..
                         } => {
-                            eprintln!("{:#}", error);
+                            print_query_error(
+                                error, statement, state.verbose_errors
+                            )?;
                         }
                         _ => eprintln!("{:#?}", e),
```

Comment:
As we're here, I'm not sure why this is debug output, not `{:#}` like everything else.

Reply:
Looks like `PrintError` (e) didn't implement `fmt::Display`.